### DISCUSSION TITLE
left block unavailable for requested intra4x4 mode -1

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#226] [Q&A] [answered]
Using version 1.9.0

On start-up I get this error 

`[2021-06-20 10:21:36] [ERROR   ] [viseron.camera.stream.fronteast] - [h264 @ 0x564c66c26dc0] left block unavailable for requested intra mode`

or

`[2021-06-20 18:17:00] [ERROR   ] [viseron.camera.stream.fronteast] - [h264 @ 0x55cbf9a20dc0] left block unavailable for requested intra4x4 mode -1`

Not sure which part of my config is causing this.

**Comment by roflcoopter** [comment, marked as answer]
If everything is working fine its nothing to worry about.
Its not directly connected to your config, its FFmpeg that prints those errors when decoding your camera.
The decoders in FFmpeg prints quite a few errors that in the end are no issues.

If you want to suppress the message you can add this to your config:
```yaml
ffmpeg_recoverable_errors:
  - "left block unavailable for requested intra"
```
